woocommerce/woocommerce
# Improve translatability of missing field notices in checkout

### All Submissions:






### Changes proposed in this Pull Request:



Closes #19418  .
As the codebase contains these fields in `class-wc-privacy-exporters.php` and `class-wc-privacy-erasers.php`, I just synced the field names with fields from `WC_Countries::get_default_address_fields` that are used in `WC_Checkout::validate_posted_data` to construct labels. 
It's a bit hackish, as `__()` should only accept string literals, but it works...

### How to test the changes in this Pull Request:

1. Switch language to a non-English one.
2. Try to make an order without filling in any info, observe incorrect translations, e.g. Slovak:
![incorrect_translation_sk](https://user-images.githubusercontent.com/2207451/42506163-5d21e776-8441-11e8-8a45-c8b746c87431.png)

3. Apply branch
4. Translate Billing First name and others correctly
5. Observer correct translations appear:
![correct_translation_sk](https://user-images.githubusercontent.com/2207451/42506498-5864b29e-8442-11e8-8901-0f47adc5fd80.png)


### Other information:




### Changelog entry

> Improve translatability of missing field notices in checkout.
